.pins/torchao-pin.txt

```diff
@@ -0,0 +1 @@
+85d03de43160328eaf350e7ec3877d3d7b57da50
```

Comment:
TODO: update to commit hash after [D62394341](https://www.internalfb.com/diff/D62394341) lands.